src/Model/PizzaConfig.java

```diff
@@ -0,0 +1,54 @@
+package Model;
+
+public class PizzaConfig {
+
+    private double base_price;
+    private OptionSet[] optionset;
+    private String size;
+    private String PizzaName;
+    private String deliveryMode;
+
+    public double getBase_price() {
+        return base_price;
+    }
+
+    public void setBase_price(double base_price) {
+        this.base_price = base_price;
+    }
+
+    public OptionSet[] getOptionset() {
+        return optionset;
+    }
+
+    public void setOptionset(OptionSet[] optionset) {
+        this.optionset = optionset;
+    }
+
+    public String getSize() {
+        return size;
+    }
+
+    public void setSize(String size) {
```

Comment:
@uwaseclarisse  do we need to do setter of each field

Reply:
ofcourse we need to do it